cpp/unit/producer.cpp

```diff
@@ -13,6 +13,14 @@
 #include "unit.h"
 #include "unit_texture.h"
 
+/**
+ * Many values in this file are hardcoded, due to limited understanding of how the original
```

Comment:
please write
```
/** @file
 * bla
 */
```
otherwise this comment will annotate the `openage` namespace :wink: